bigbluebutton-html5/imports/ui/components/chat/chat-graphql/queries.ts

```diff
@@ -0,0 +1,44 @@
+import { gql } from '@apollo/client';
+
+export const IS_TYPING_PUBLIC_SUBSCRIPTION = gql`subscription IsTyping($chatId: String!) {
+  user_typing_public(
+      limit: 4,
+      where: {
+        isCurrentlyTyping: {_eq: true}
+        chatId: {_eq: $chatId}
```

Comment:
I don't think using the chatId is interesting in this case because the view already filters for public chat